js/bruh-dash.js

```diff
@@ -1,83 +1,297 @@
 var global = window || GLOBAL;
 
+
 global.bruhdash = {
-  chunk: function(){
 
+  chunks:chunks,
+
+  chunk: function(array, number){
+    if(isNaN(number))
+      return false;
+    if(number < 1)
+      throw new RangeError('Parameter must be at least' + 1);
+    var bag = [];
+    var marker = 0;
+    while(marker < array.length){
+      var cut = array.slice(marker, (marker + number));
+      bag.push(cut);
+      marker+= number;
+    }
+
+    return bag;
   },
 
-  compact: function() {
-
+  compact: function(array) {
+    var clean = [];
+    for(var i = 0; i < array.length; i++)
+      if(array[i] !== 0 && array[i] !== null && array[i] !== '' && array[i] !== false && !isNaN(array[i]))
```

Comment:
You could leverage truthiness values to clean up this line to be a single conditional check.
💁